README.md

```diff
@@ -22,12 +22,12 @@ For example:
     export GZ_VERSION=fortress
 
 > You only need to set this variable when compiling, not when running.
```

Comment:
nit, Looking at the preview, I would recommend removing the leading `>` instead of adding it to the rest of the lines